moveit_setup_assistant/src/widgets/setup_assistant_widget.cpp

```diff
@@ -50,11 +51,14 @@
 #include <QDir>
 #include <pluginlib/class_loader.hpp>  // for loading all avail kinematic planners
 // Rviz
+DIAGNOSTIC_PUSH
+SILENT_UNUSED_PARAM
 #include <rviz/render_panel.h>
 #include <rviz/visualization_manager.h>
 #include <rviz/view_manager.h>
 #include <rviz/default_plugin/view_controllers/orbit_view_controller.h>
 #include <moveit/robot_state_rviz_plugin/robot_state_display.h>
```

Comment:
How is the unused param calculated? We should be able to fix the warning if it's in our own file, right?

Reply:
What do you mean by _calculated_? This specific macro disables unused parameter warnings.
For other warnings, one should add another specific macro.

Reply:
Sorry, calculated was a poor choice of words. I mostly wanted to know if we can fix the warning for `robot_state_display.h`, or if it needs be be silenced because how it's used (or not used) in this particular file.